python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -1130,17 +1157,30 @@ def logs_after(
         of_type: Optional["DagsterEventType"] = None,
         limit: Optional[int] = None,
     ):
-        return self._event_storage.get_logs_for_run(
-            run_id,
-            cursor=cursor,
-            of_type=of_type,
-            limit=limit,
-        )
+        from dagster.core.events import DagsterEventType
+
+        # We remove REGISTER_RUN_ASSET events here because these events
```

Comment:
This feels a little low-level to do the filtering here.  What do you think about letting these pass through the instance / storage APIs, and just filtering either in the graphql or in the JS layer?